metrics/nist/requirements.txt

```
@@ -0,0 +1,3 @@
+git+https://github.com/huggingface/evaluate@a45df1eb9996eec64ec3282ebe554061cb366388
+datasets~=2.0
```

Just merged a PR that updates this:
```suggestion
git+https://github.com/huggingface/evaluate@{COMMIT_PLACEHOLDER}
```